cartoframes/utils/metrics.py

```diff
@@ -87,9 +97,10 @@ def build_metrics_data(event_name, extra_metrics_data):
 
 
 @silent_fail
-def post_metrics(event_name, extra_metrics_data):
-    json_data = build_metrics_data(event_name, extra_metrics_data)
-    result = requests.post('https://bmetrics.cartodb.net', json=json_data, timeout=2)
+def post_metrics(event_name, extra_metrics_data, server_domain_tld):
+    metrics_server = STAG_METRICS_SERVER if server_domain_tld == STAG_DOMAIN_TLD else PROD_METRICS_SERVER
```

Comment:
I would move this logic to `utils.get_metrics_server(server_domain_tld)` or `get_metrics_server(server_domain_tld)` (this file)